README.md

```diff
@@ -1,71 +1,34 @@
-    ```
-    python usage.py
-    ```
-7. Create a production build and publish:
-    1. Build your code:
-    ```
-    npm run build:js
-    npm run build:py
-    ```
-    2. Create a Python tarball
-    ```
-    python setup.py sdist
-    ```
-    This distribution tarball will get generated in the `dist/` folder
-
-    3. Test your tarball by copying it into a new environment and installing it locally:
-    ```
-    pip install my_dash_component-0.0.1.tar.gz
-    ```
+## Usage
 
-    4. If it works, then you can publish the component to NPM and PyPI:
-    ```
-    npm run publish
-    ```
-    ```
-    twine upload dist/dash_component-0.0.1.tar.gz
-    ```
-8. Share your component with the community! https://community.plot.ly/c/dash
-
-# More details
-- Include CSS files in your distribution folder (`my_dash_component`) and reference them in `MANIFEST.in`
-- The `tests` folder contains a sample integration test. This will run a sample Dash app in a browser. Run this with:
-    ```
-    python -m tests.test_render
-    ```
-    The Dash team uses these types of integration tests extensively. Browse the Dash component code on GitHub for more examples of testing (e.g. https://github.com/plotly/dash-core-components)
-- Publishing your component to NPM will make the JavaScript bundles available on the unpkg CDN. By default, Dash servers the component library's CSS and JS from the remote unpkg CDN, so if you haven't published the component package to NPM you'll need to set the `serve_locally` flags to `True`. We will eventually make `serve_locally=True` the default, [follow our progress in this issue](https://github.com/plotly/dash/issues/284).
-- Watch the [component boilerplate repository](https://github.com/plotly/dash-component-boilerplate) to stay informed of changes to our components.
+To use this boilerplate:
 
+1. Install the requirements:
+    ```
+    $ pip install cookiecutter
+    $ pip install virtualenv
+    ```
+   [Node.js/npm is also required.](https://nodejs.org/en/)
+2. Run cookiecutter on the boilerplate repo:
+    ```
+    $ cookiecutter git@github.com:plotly/dash-component-boilerplate.git
+    ```
+3. Answer the questions about the project.
+    - project name: Clean project name, can contains spaces and special characters.
+    - component name: derived from project_name, lowercase, spaces and '-' are replaced by '_', ascii only.
+    - author info: author_name and author_email for package.json metadata.
+    - description: the project description, included in package.json.
+    - version: The initial version.
+    - license: License type for the component lib.
+    - publish_on_npm: Set to false to only serve locally from the package data.
+    - install_dependencies: Set to false to only generate the project structure.
+4. The project will be generated in the folder of `project_shortname`.
+5. Follow the directions in the generated README to start developing your new Dash component.
```

Comment:
Could we create a little ascii-cenima recording of this process and include it here? https://asciinema.org/

Reply:
I'd like to but it's not for windows, if someone wants to do it after merge and include it, I'll create an issue for it.